### DISCUSSION TITLE
Nested Editors (editors inside a Node View): is that allowed?

### DISCUSSION THREAD

**Discussion by reivilibre** [ueberdosis/tiptap#3400] [General Questions]
I found out that you can't directly have multiple content slots in a node view.

I thought that putting Tiptap editors inside the node view might be OK as a workaround — for collaboration purposes, they could be other Y.js trees within the same Y.js document.

That said, the idea sounds like it is a bit dodgy. Does anyone know whether it's safe to do this?

**Comment by Priestch** [comment]
@reivilibre Of course, you can.

See the official prosemirror [embedded code editor](https://prosemirror.net/examples/codemirror/) example, be cautious, it may be buggy.
